From 8577a73f050d75ee4762da9a14dfece86cd3add9 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Tue, 30 Apr 2024 15:00:01 +0000
Subject: [PATCH] disable metadata_csum_seed and orphan_file

These have been enabled in v1.47.0 by default
but the UEFI firmware does not support them and
fails to mount the root filesystems, causing the
kernel and dtb to fail to load.


Upstream-Status: Inappropriate
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 misc/mke2fs.conf.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/misc/mke2fs.conf.in b/misc/mke2fs.conf.in
index b7fc95df..05680992 100644
--- a/misc/mke2fs.conf.in
+++ b/misc/mke2fs.conf.in
@@ -11,7 +11,7 @@
 		features = has_journal
 	}
 	ext4 = {
-		features = has_journal,extent,huge_file,flex_bg,metadata_csum,metadata_csum_seed,64bit,dir_nlink,extra_isize,orphan_file
+		features = has_journal,extent,huge_file,flex_bg,metadata_csum,64bit,dir_nlink,extra_isize
 	}
 	small = {
 		blocksize = 1024
-- 
2.17.1

